vmdb/app/controllers/application_controller/explorer.rb

```diff
@@ -927,8 +927,7 @@ def x_get_tree_custom_kids(object, options)
       return count_only ? objects.length : objects
     when :alert_profile
       # Add all alert profiles so links back from Alerts etc will work - TODO: figure out how to load on demand
-      objects = MiqAlertSet.all(:conditions=> ["(mode = ?)", object[:id].split('-')],
-                          :order => "lower(description) ASC")
+      objects = MiqAlertSet.where(:mode => object[:id].split('-')).order("lower(description) ASC")
```

Comment:
> Agreed, @dclarizio is this method on the TODO list for rewrite and/or test writing? 
@jrafanie Yes, the treebuilder class is being refactored and extended as we head towards replacing the dhtmxltrees.